### DISCUSSION TITLE
Notify users that the maximum number of input items has been reached, or cause a panic

### DISCUSSION THREAD

**Discussion by sehkone** [aicers/frontary#418] [Ideas]
Frontary defines a maximum number of input items allowed in each layer of the item hierarchy. Therefore, when an item is added to a `Group` item, the user should be notified, and the addition should be prevented. Conversely, for `VecSelect`, `Checkbox`, or `Radio` items, where the number of children is statistically determined by Frontary users, the program should trigger a panic if the maximum limit is reached.

**Comment by octoaide** [comment]
Hi @sehkone, this issue has been open and assigned to you for quite some time without recent activity. Could you please provide an update on its status or if you foresee any progress on implementing the defined logic for maximum input items?

**Comment by sehkone** [comment]
@octoaide, can you take this for me?

**Comment by sehkone** [comment]
I think I need to specify implementation details or do this myself. Since neither is not urgent and the current code is safe enough without the feature, I am moving this to discussion.
